src/Raven.Server/Documents/Replication/IncomingReplicationHandler.cs

```diff
@@ -1179,6 +1189,14 @@ protected override long ExecuteCmd(DocumentsOperationContext context)
                                 if (tombstone != null && ChangeVectorUtils.GetConflictStatus(item.ChangeVector, tombstone.ChangeVector) == ConflictStatus.AlreadyMerged)
                                     continue;
 
+                                // tombstone structure means doc-id | REC-SEP | ...
+                                var keySpan = attachmentTombstone.Key.AsReadOnlySpan();
+                                int endOfDocumentId = keySpan.IndexOf(SpecialChars.RecordSeparator);
+                                Debug.Assert(endOfDocumentId != -1);
+                                string documentId = Encoding.UTF8.GetString(keySpan[..endOfDocumentId]);// will throw if no separator found 
```

Comment:
so this will end up in a replication loop, right? is it a valid scenario?

Reply:
It will _not_ be a replication loop. It will send this once, yes.
It's an update (merged with the old change vector), it will be sent to the other side, where it will be _older_ than the previous version, and then discarded.